lib/grbl/src/credentials.hpp

```diff
@@ -0,0 +1,3 @@
+#define WIFI_HOSTNAME "printer"
```

Comment:
If you add the preprocessor lines then it's good to be merged. Something like this:

#ifndef credentials_h
#define credentials_h

#define WIFI_SSID "Whatever"
#define WIFI_PASSWORD "mydummypassword"

#define WIFI_HOSTNAME "grblesp"

#endif

Reply:
Done